pkgs/development/python-modules/python3-msrplib/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k, python3-application
+, python3-gnutls, python3-eventlib, twisted, zope_interface, ... }:
+
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.21.1";
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-msrplib";
+    rev = "5bd069620d436d5a65e1c369e43cc6b88857fb9e";
```

Comment:
```suggestion
    rev = version;
```

Reply:
This tag does not exist. It should be created, as for python3-gnutls.
I have sent a message to the developer's mailing list about several of these repos not having tags for the most recent versions, and will send them darcs patches if they accept (the GitHub repositories are mirrors of their darcs repos, I believe, and they request contributions via darcs [here](https://sipsimpleclient.org/contribute/)).